src/main.go

```diff
@@ -118,6 +123,30 @@ func Cmd() *cobra.Command {
 		"encrypted gas limit",
 	)
 
+	cmd.PersistentFlags().BoolVarP(
+		&Config.MetricsConfig.Enabled,
+		"metrics-enabled",
+		"",
+		false,
+		"to enable promnetheus metrics",
+	)
+
+	cmd.PersistentFlags().StringVarP(
```

Comment:
would be good to set the default values here for host and port to the default prometheus ones? local host and 9090?